src/datasets/mesh/mod.rs

```diff
@@ -61,8 +61,11 @@ use crate::Generate;
 #[cfg_attr(feature = "nanoserde", derive(nanoserde::SerBin, nanoserde::DeBin))]
 #[cfg_attr(feature = "wiring", derive(Wiring, Unwiring))]
 pub struct Vector3 {
+    #[cfg_attr(feature = "wiring", concat_start)]
```

Comment:
[nonblocking] Suggestion: a `concat_all` attribute on the type could be a nice way to tidy some of these structs up.

Reply:
Agreed, the ultimate plan to remove them all by making derive macro smarter.

Please put this merge on hold, I will see if I can tackle it tomorrow or during the weekend.